Makefile

```diff
@@ -0,0 +1,46 @@
+SHELL=/usr/bin/env bash -o pipefail
+
+# Image URL to use all building/pushing image targets
+IMG ?= quay.io/observatorium/observatorium-operator:latest
+BIN_DIR ?= $(shell pwd)/tmp/bin
+
+CONTROLLER_GEN ?= $(BIN_DIR)/controller-gen
+
+# Install CRDs into a cluster
+install: manifests
+	kubectl apply -f deploy/crds
+
+# Uninstall CRDs from a cluster
+uninstall: manifests
+	kubectl delete -f  deploy/crds
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests: $(CONTROLLER_GEN)
+	$(CONTROLLER_GEN) crd paths="./..." output:crd:artifacts:config=deploy/crds
+
+# Run go fmt against code
+fmt:
+	go fmt ./...
+
+# Run go vet against code
+vet:
+	go vet ./...
+
+# Generate code
+generate: $(CONTROLLER_GEN)
+	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+
+# Build the docker image
+docker-build:
+	docker build . -t ${IMG}
```

Comment:
We need to add this to the circle ci config so that images are built in an isolated env rather than by hand on a Deb's machine. Also, we avoid using `latest`. We should use a unique tag per commit/date eg https://github.com/observatorium/up/blob/master/.drone.yml#L52

Reply:
Done, following https://github.com/observatorium/observatorium/blob/master/.circleci/config.yml